tools/test_set_firewall_ns.py

```diff
@@ -0,0 +1,43 @@
+#! /usr/bin/env python3
+
+# stdlib
+import sys
+import json
+# local
+from cloudcix_primitives import set_firewall_ns
+
+cmd = sys.argv[1] if len(sys.argv) > 1 else None
+set_name = "IE_41"
+namespace_name = "ns1100"
+set_type = 'ipv4_addr'
+set_elements = '8.8.8.8, 1.1.1.1, 192.168.0.1/24'
+config_file = "/etc/cloudcix/pod/configs/config.json"
+
+if len(sys.argv) > 2:
+    namespace_name = sys.argv[2]
+if len(sys.argv) > 3:
+    set_name = sys.argv[3]
+if len(sys.argv) > 4:
+    set_type = sys.argv[4]
```

Comment:
I ran this a couple of times when I used it for testing geo_a_firewall_ns and something irked me...I finally realized that the set_type parameter is redundant for the `update()` function. I created my own local patch for getting rid of it: https://github.com/jgrassler/primitives/commit/c0633ce857e774223e0c46dee74c5370f560bbe4 (feel free to use).